cron/centreon-backup.pl

```diff
@@ -585,6 +578,11 @@ ()
 			print STDERR "Unable to copy PHP configuration file\n";
 		}
 	}
+	# PHP configuration ini files
+	`cp -pr /etc/opt/rh/rh-php71/php.d/ $TEMP_CENTRAL_ETC_DIR/php/`;
```

Comment:
I'm not a big fan of hardcoded paths. People that are not using rh-php71 will have issues using Centreon Backup. For me this is a no go.

Is there any other way to backup the php.d directory ?

Reply:
If php binary is know by system:

`PHP=$(php --info | grep "Additional .ini files parsed" | cut -d '>' -f2 | xargs dirname); echo $PHP`

If $PHP is empty, maybe make a list of test to perform (rh-php71, php-7.x-remi, ...) or try if locate exists to run PHP$(locate php.ini | egrep "php.ini$" | xargs dirname)